lesson06/assignment/Drama.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL" crossorigin="anonymous"></script>
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+        <link rel="stylesheet" href="css/style.css" />
+        <title>Assignment 06</title>
+    </head>
+    <body>
+        <header>
+          <div class="grid-container">
+                <div class="header-left">
+                    <div class="header-left-1">
+                      <h1>BOOK MY EVENT</h1>   
+                    </div>
+                    <div class="header-left-2">
+                      <form method="get">
+                        <input type="text" name="q" placeholder="search for events">
+                        <input type="submit" value="search">
+                      </form>
+                    </div>
+                </div>
+                <div class="header-right">
+                    <label>Location<br>
+                        <select name="location">
+                            <option value="Seattle">Seattle</option>
+                            <option value="Bellevue">Bellevue</option>
+                            <option value="Tacoma">Tacoma</option>
+                        </select>
+                    </label>
+                </div>
+            
+     
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
Your navbar doesn't show up for me on mobile/small screens.  You might try copying and pasting an example from the docs and then editing it to match your links.

https://getbootstrap.com/docs/5.0/components/navbar/